### DISCUSSION TITLE
When dotnet needs to be installed. Best practice?

### DISCUSSION THREAD

**Discussion by axtens** [unoplatform/uno#14588] [Q&A]
The user attempts to install the application but doesn't have the a dotnet runtime installed or the correct one installed. What's the usual approach to this? Is/can the installer be bundled with the installer? Can the installer walk the user through the download and install? What's the best practice?

**Comment by jeromelaban** [comment]
Thanks for the question.

Which platform are you targeting? If it's windows, have you taken a look at [this deployment document](https://learn.microsoft.com/en-us/windows/apps/package-and-deploy/deploy-overview)?

**Comment by limeniye** [comment]
If you are using the .net 8.0 you can publish your application with AOT technology.
Such a result file does not require additional .net libraries, but it has more weight.
https://learn.microsoft.com/en-us/dotnet/core/deploying/native-aot/?tabs=net7%2Cwindows
